common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java

```diff
@@ -408,6 +408,12 @@ private ErrorStrings() {
                     "Please make sure to use your organizational account. " +
                     "If that doesn’t help, please return the device to your administrator.";
 
+    /**
+     * Device Code Flow only.
+     * Authorization has not been completed yet.
+     */
+    public final static String DEVICE_CODE_FLOW_AUTHORIZATION_PENDING_CODE = "authorization_pending";
```

Comment:
```suggestion
    public final static String DEVICE_CODE_FLOW_AUTHORIZATION_PENDING_ERROR_CODE = "authorization_pending";
```

Reply:
Looks like we've followed the pattern `....ERROR_CODE` and `....ERROR_MESSAGE` but I don't see that being used in these new constants. Let's try to be consistent.

Reply:
Will do!